pom.xml

```diff
@@ -98,7 +98,7 @@
         <!-- Container images -->
         <quarkus.container-image.registry>quay.io</quarkus.container-image.registry>
         <quarkus.container-image.group>ecosystem-appeng</quarkus.container-image.group>
-        <quarkus.container-image.tag>1.0.2</quarkus.container-image.tag>
+        <quarkus.container-image.tag>v1.0.2</quarkus.container-image.tag>
```

Comment:
If you change the container image tag, you have to update all the references to the image in the examples as well (e.g. party-routing-profile-example:1.0.2 -> party-routing-profile-example:v1.0.2)